metadata_service/config.py

```diff
@@ -29,6 +32,10 @@ class Config:
 
     PROXY_USER = os.environ.get('CREDENTIALS_PROXY_USER', 'neo4j')
     PROXY_PASSWORD = os.environ.get('CREDENTIALS_PROXY_PASSWORD', 'test')
+    PROXY_ENCRYPTED = True
```

Comment:
@davcamer  little late to the party. But is there a reason this is not controlled with env vars ?